webviz_config/templates/webviz_template.py.jinja2

```diff
@@ -151,3 +155,6 @@ if __name__ == "__main__":
         dev_tools_silence_routes_logging=False,
       {% endif %}
     )
+else:
+    if use_oauth2:
```

Comment:
add a comment that this is only necessary when not running on localhost, as it is implemented as part of the `LocalhostToken`?